plugins/woocommerce-admin/client/launch-your-store/hub/styles.scss

```diff
@@ -15,6 +15,11 @@
 			flex-direction: row;
 			background-color: #fcfcfc;
 
+			.customize-your-store-edit-site-resizable-frame__inner-content {
+				height: 100%;
+				border-radius: 20px !important;
```

Comment:
Just curious, is this necessary? I toggle the border-radius on and off, but it doesn't make a difference. 🤔 

https://github.com/woocommerce/woocommerce/assets/4344253/b6997ce9-beed-4caf-b311-3057a4b55785

Reply:
I noticed that as well. Since it had the `!important` tag on I left it, but now that you mention it, I will just remove it